src/base/_typography.scss

```diff
@@ -13,17 +13,3 @@ p {
   line-height: $base-line-height;
   font-size: $base-font-size;
 }
-
-a {
-  color: inherit;
-  transition: all $base-transition-time;
-  opacity: 0.75;
-  font-weight: bold;
```

Comment:
These styles are too specific, I don't want to undo them in components like AeListItem one.